packages/cozy-procedures/src/Procedure.jsx

```diff
@@ -5,24 +5,30 @@ import get from 'lodash/get'
 import store from './redux/store'
 import Context from './redux/context'
 import creditApplicationTemplate from './templates/creditApplicationTemplate'
-import { init as initPersonalData } from './redux/personalDataSlice'
+import {
+  init as initPersonalData,
+  fetchMyself
+} from './redux/personalDataSlice'
 import withLocales from './withLocales'
+import { withClient } from 'cozy-client'
 
 class Procedure extends React.Component {
   componentDidMount() {
     this.props.initPersonalData(
       get(creditApplicationTemplate, 'personalData.schema.properties', {})
     )
+    this.props.fetchMyself(this.props.client)
   }
 
   render() {
     return this.props.children
   }
 }
 
-const mapDispatchToProps = dispatch => ({
-  initPersonalData: template => dispatch(initPersonalData(template))
-})
+const mapDispatchToProps = {
```

Comment:
can we have a separate file for the container ?

Reply:
It's done but I also had to put `ProcedureWithStore` in another file (named it `App`, not sure it's the best name)